codalab/apps/web/templates/web/competitions/view.html

```diff
@@ -276,8 +276,12 @@ <h3 class="panel-title"><strong>Competition Ends</strong></h3>
                 </div>
             {% endif %}
         </div>
-        <div class="col-md-8">
-            {% include "web/competitions/_chart.html" %}
+        <div align="center" class="col-md-8">
+            {% if active_phase.is_blind %}
+                <i>This phase is blind</i>
```

Comment:
Do we even need a message here if the phase is blind?

Reply:
Nope I guess not